File: exercicios_python/exercicios/exercicio1.py
# Crie uma função que dado dois valores (passados como parâmetros) mostre no console a soma, subtração, 
# multiplicação e divisão desses valores. 

def operacoes_basicas(a, b):
  soma = a + b
  subtracao = a - b
  multiplicacao = a * b
  if b != 0:
    divisao = a / b
  else:
    divisao = "Indefinido (divisão por zero)"
  
  print(f"Soma: {soma}")
  print(f"Subtração: {subtracao}")
  print(f"Multiplicação: {multiplicacao}")
  print(f"Divisão: {divisao}")
  print("----------------------------------")
  print(" ")




operacoes_basicas(50, 25)
operacoes_basicas(10, 2)

File: exercicios_ruby/exercicios/exercicio1.rb
# Crie uma função que dado dois valores (passados como parâmetros) mostre no console a soma, subtração, multiplicação e divisão desses valores.

def operacoes(a, b)
  soma = a + b
  subtracao = a - b
  multiplicacao = a * b
  if a == 0
    divisao = "Divisão por Zero não é aceita!"

  else
    divisao = a / b.to_f
  end

  puts "Resultado Soma: #{soma}"
  puts "Resultado Subtração: #{subtracao}"
  puts "Resultado Multiplicação: #{multiplicacao}"
  puts "Resultado Divisão: #{divisao}"

end

operacoes(5, 2)